### DISCUSSION TITLE
Avast (virus scanner) complained when WinEdt and WinEdtPDF wanted to rename a folder / change or delete a file

### DISCUSSION THREAD

**Discussion by tozGitHub** [WinEdt-Team/WinEdt-Support#28] [2.2 Issues, Bugs and Problems]
Dear Alex, dear community,

My virus scanner complained, but proposed an easy fix when WinEdt and WinEdtPDF wanted to change folders / files.
![4dd60088ffbc9befc93adafb0c039cce_MD5](https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/73b7c50a-e8c4-4d69-a10c-a1b0d4f36fad)
![a25d9a987fb007fd301b1f1fcffe44aa_MD5](https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/188eb635-b3da-4388-86ed-2b29e0ee7826)

Windows version: 10.0.22631 Build 22631. WinEdt Build: 20240317  (v. 11.2) - 64-bit (the portable version; I did not run into this with the newer build, but this probably because I moved the old version to a different folder, and put the new version into the old folder).

Best,
Thomas

**Comment by Alex-Simonic** [comment]
With the portable version (or even the installer version) warnings like this are always possible. I have done all I could by signing all binaries with "WinEdt Inc." code signing certificate compliant with the latest security standards (a costly proposition at $629 per year from my pocket). Without signed binaries, Windows blocked me from downloading the zip or exe file from our own website and after I forced the downloaded zip file to be opened the unsigned WinEdtPDF.exe was completely removed from all my folders by Windows Defender with all kinds of (bogus) security threat alarms. And yet people somehow manage to contaminate their computers. Go figure...
